### PR TITLE
WPF panel

### PR DESCRIPTION
**Pull request by aydevelop**

### CODE REVIEW

**Review by Anatolii-Ivanov — COMMENTED** [review]
I don't think you need [RecodoDesktop/RecodoDesktop/images_origin/image.png]
And why you need so much colors of the same icon?
+ So much files of the same font? Let's use Resular, Semibold, Italic - this should be enough

**Comment by Jyuart** [comment]
Do we really need [image.png](https://github.com/BinaryStudioAcademy/bsa-winter21-22-recodo/pull/8/files#diff-49c8d85343a87dc60dd28c055185316c68032972b1f033917c3ddc6f196c87f4)?

File: RecodoDesktop/RecodoDesktop/MainWindow.xaml.cs [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
-using System.Windows.Media.Imaging;
-using System.Windows.Navigation;
-using System.Windows.Shapes;
 
 namespace RecodoDesktop
 {
-    /// <summary>
-    /// Interaction logic for MainWindow.xaml
-    /// </summary>
     public partial class MainWindow : Window
```
Comment (Jyuart) [inline RecodoDesktop/RecodoDesktop/MainWindow.xaml.cs]: This should go to another window, I suppose

Comment (aydevelop) [reply RecodoDesktop/RecodoDesktop/MainWindow.xaml.cs]: MainWindow.xaml.cs designed for the logic of working with MainWindow.xaml

Comment (Jyuart) [reply RecodoDesktop/RecodoDesktop/MainWindow.xaml.cs]: I mean that the panel you're working on is not a main window of the application, don't you think?

Comment (Anatolii-Ivanov) [reply RecodoDesktop/RecodoDesktop/MainWindow.xaml.cs:6]: It's fine, let it stay as it is for now

**Review by Anatolii-Ivanov — APPROVED** [review]
For me, it's fine

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]
Oh, I understand, yes, please stay mainWindow as it is, the control  form won't be opened by default, so move it to separate xaml+cs files (For example ControlForm.xaml) (For now you can add it to show on startup (check Vasylii's PR))

File: frontend/.vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,25 @@
 {
```
Comment (Anatolii-Ivanov) [inline frontend/.vscode/settings.json]: Do you need this?

Comment (aydevelop) [reply frontend/.vscode/settings.json]: Deleted!

**Review by Jyuart — APPROVED** [review]

**Pull request closed without merging**